content/posts/quickstart/index.fr.md

```diff
@@ -152,9 +152,9 @@ git push origin HEAD
 
 ### Et ensuite ?
 
-- Personnaliser l'arrière-plan, le logo, et quelques autres choses de votre site en suivant [ce guide](/posts/configuration/site-parameters/).
-- Ajouter des informations sur vous en suivant [ce guide](/posts/configuration/sections/about/).
-- Ajouter les informations sur vos compétences en suivant [ce guide](/posts/configuration/sections/skills/).
-- Ajouter les informations sur vos expériences en suivant [ce guide](/posts/configuration/sections/experiences).
-- Déployer votre site sur Github Page en suivant le guide par [ici](/posts/getting-started/github-pages/).
-- Déployer votre site sur Netlify en suivant le guide par [ici](/posts/getting-started/netlify/).
+- Personnaliser l'arrière-plan, le logo, et quelques autres choses de votre site en suivant [ce guide](/posts/configuration/site-parameters/index.fr.md).
+- Ajouter des informations sur vous en suivant [ce guide](/posts/configuration/sections/about/index.fr.md).
+- Ajouter les informations sur vos compétences en suivant [ce guide](/posts/configuration/sections/skills/index.fr.md).
+- Ajouter les informations sur vos expériences en suivant [ce guide](/posts/configuration/sections/experiences/index.fr.md).
+- Déployer votre site sur Github Page en suivant le guide par [ici](/posts/getting-started/github-pages/index.fr.md).
+- Déployer votre site sur Netlify en suivant le guide par [ici](/posts/getting-started/netlify/index.fr.md).
```

Comment:
They don't renders as valid link. Please don't specify `index.fr.md` part. Instead add `fr` prefix in the URL. For example:

```
- Personnaliser l'arrière-plan, le logo, et quelques autres choses de votre site en suivant [ce guide](/fr/posts/configuration/site-parameters/).
```

Reply:
OK so, it's fixed. But I have a problem with that. When you navigate on this page in french version, if you want go to another page. You will arrive on the target but English version.

Reply:
I think the link should be `/fr/posts/configuration/sections/experiences/`.
At least, in my case works.

Reply:
@BernatBC Oh, OK, I will test this solution now. Thank.

Reply:
You're right @BernatBC, it work fine. Thank you so much for your solution.

Reply:
@hossainemruz you can merge, it's OK now.

Reply:
Thank you for fixing the issues.